src/main/java/micdoodle8/mods/galacticraft/core/inventory/SlotBuggyBench.java

```diff
@@ -68,12 +69,12 @@ public boolean isItemValid(ItemStack itemStack) {
                     && itemStack.getItemDamage() == 100;
         } else if (this.index >= 4 && this.index <= 7) {
             return itemStack.getItem() == GCItems.partBuggy && itemStack.getItemDamage() == 0;
-        } else if (this.index >= 8 && this.index <= 11) {
+        } else if (this.index >= 8 && this.index <= 11 && Loader.isModLoaded(Constants.MOD_ID_GREGTECH)) {
```

Comment:
just checking for gregtech also kicks in for GT6.

If you want to check for GTNH better check for dreamcraft, if you specifically want to check for GT5U-GTNH, check for gregtech but not gregapi. see for example https://github.com/GTNewHorizons/ThaumicBases/pull/36